packages/components/src/components/radio-group/radio-group.stories.ts

```diff
@@ -41,217 +48,130 @@ export default {
+        <label slot="label">Horizontal Group</label>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
  * Use the `required` attribute to mark the element as required. This can be used for form validation purposes.
+ *
+ *
+ * **Accessibility:** To ensure screen-reader compatibility, consider including a statement such as "Fields marked with an asterisk (*) are required" at the start of the form.
  */
 
 export const Required = {
-  parameters: { controls: { exclude: ['required'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: [{ type: 'attribute', name: 'size' }],
-        y: { type: 'attribute', name: 'required' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the `form-control`, `form-control-label` and `form-control-input` part selectors to customize the radio-group.
- */
-export const Parts = {
-  parameters: {
-    controls: { exclude: ['form-control', 'form-control-label', 'form-control-input'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-radio-group::part(...){outline: solid 2px red}',
-          values: ['form-control', 'form-control-label', 'form-control-input'].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-radio-group::part(${part}){outline: solid 2px red}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      constants: [{ type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' }],
-      args
-    });
-  }
-};
-
-/**
- * sd-radio-group are fully accessibile via keyboard.
- */
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-radio-group');
-    await waitUntil(() => el?.shadowRoot?.querySelector('label'));
-
-    if (el?.shadowRoot) {
-      const label = el.shadowRoot.querySelector('label');
-      if (label) {
-        await userEvent.type(label, '{space}', { pointerEventsCheck: 0 });
-      }
-    }
-  }
-};
-
-/**
- * This shows `sd-radio-group` using multiple `sd-radio-buttons` in it's default state.
- */
+  name: 'Required',
+  render: () => html`
+    <form id="required-form" class="flex flex-col gap-2">
+      <sd-radio-group name="radio-group" required boldlabel>
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Required Group</label>
+      </sd-radio-group>
+
+      <div class="flex gap-2">
+        <sd-button class="w-min" type="submit">Submit</sd-button>
+        <sd-button class="w-min" type="reset" variant="secondary">Reset</sd-button>
+      </div>
+    </form>
```

Comment:
Can you add more space between radio group and the button?
<img width="332" alt="image" src="https://github.com/user-attachments/assets/e88e2797-3775-4736-9566-02a3a406a764">